flybywire-aircraft-a320-neo/Business.json

```
@@ -0,0 +1,6 @@
+{
+  "PriceInUSD": 0,
+  "ThirdPartyShortName": "flybywiresimulations",
```

```suggestion
  "ThirdPartyShortName": "flybywire",
```
Should change to that to match our package name